modules/ROOT/partials/integrations/blazor-tech-ref.adoc

```diff
@@ -19,7 +19,7 @@ The `+TinyMCE.Blazor+` `+Editor+` component accepts the following properties:
   JsConfSrc="path_to_jsObj"
   Conf="@yourConf"
   ApiKey="no-api-key"
-  LicenseKey="your-license-key"
+  LicenseKey="gpl" // gpl for open source, T8LK:... for commercial
```

Comment:
```suggestion
  LicenseKey="gpl"
```
* [ ] Move this comment to below the example, as it's not syntactically valid where it is